plutus-chain-index/README_alpha-release_Marconi.md

```diff
@@ -0,0 +1,75 @@
+![Marconi-Logo_sm](https://user-images.githubusercontent.com/104791413/178748103-077ed7e0-6071-461f-b9ed-97b3b7bd5b24.png)
+
+# Marconi Alpha Release
+
+## The Cardano blockchain indexer for dApp developers
+
+Marconi is a lightweight, customizable solution for dApp developers who need to index and query the Cardano blockchain.
+
+## Introduction
+
+Marconi is a blockchain query solution that you can configure for the types of information that you need to index and according to how you need to structure your database schema. Marconi provides a way for Plutus off-chain code to have access to data that has already been stored on-chain by indexing portions of the blockchain. 
+
+Marconi gives you the ability to customize the set of data that is indexed. It provides a library of functionality with which you can do the following:
+* build your own indexers, 
+* work only with the data that you care about for your application, 
+* use whatever storage engine you prefer, and 
+* support only the queries that you need to support.
+
+### Scalability through Customized Solutions for Each Use Case
+
+The philosophy behind Marconi is to maintain scalability by customizing the solution for each use case. You can filter the streams of information to transmit only the data you need for your dApp, minimizing network traffic and database storage. You can also customize database schemas and queries so that the dApps have the most efficient and scalable application-specific API to interact with the data.
+
+### Indexing solution ### 
+
+Any indexing solution must deal with blockchain rollbacks, which can happen as the network achieves consensus. Marconi is designed to deal with rollbacks as elegantly as possible while simultaneously finding an effective balance between memory, disk and CPU usage. 
+
+Managing rollbacks is very simple and fast. Marconi drops the events that were rolled back. No complicated logic is required to undo the projection of the list of events on disk, which Marconi would need if it stored everything on disk as quickly as possible. 
```

Comment:
```suggestion
Managing rollbacks is very simple and fast. Marconi drops the events that were rolled back. Because the events that can be rolled back are stored in memory, no complicated logic is required to undo the projection of the list of events on disk, which Marconi would need if it stored everything on disk as quickly as possible. 
```